### PR TITLE
added image to the footer to increase height :P

### PR DESCRIPTION
**Pull request by karanasthana**

### CODE REVIEW

File: src/Sections/Footer.js [hunk 1]
```diff
@@ -2,40 +2,50 @@ import React from 'react';
...
-                    </a>
-                </Button>
-            </div>
-            <div className="footer-icons-row">
-                <div className="copyright-container">
-                    Copyright <i className ="ri-copyright-line" /> {' Karan Asthana'} All rights reserved.
+            <div className="container">
+                <div className="pdf-row">
+                    <Button primary>
+                        <a href={CONSTANTS.RESUME_PDF_LINK} target="_blank" rel="noreferrer" download style={{  }}>
```
Comment (ayu-ayushisood) [inline src/Sections/Footer.js]: why is style attribute empty?

File: src/Sections/sass.scss [hunk 2]
```diff
@@ -57,18 +57,25 @@
...
             display: flex;
-            justify-content: center;
+            flex-direction: column;
+            justify-content: flex-start;
             color: white;
+
+            .footer-img-karan {
+                max-height: 25vh;
+                width: auto;
+                max-width: 25vh;
```
Comment (ayu-ayushisood) [inline src/Sections/sass.scss:70]: width should be in vw imo

**Review by ayu-ayushisood — CHANGES_REQUESTED** [review]

File: src/Sections/sass.scss (hunk 2 above)
Comment (karanasthana) [reply src/Sections/sass.scss:70]: This is intentional

File: src/Sections/Footer.js (hunk 1 above)
Comment (karanasthana) [reply src/Sections/Footer.js]: Will remove

**Pull request merged**